slice/internal/controller/workload_controller.go

```diff
@@ -433,6 +434,11 @@ func (r *WorkloadReconciler) syncSlices(ctx context.Context, wl *kueue.Workload,
 	createdSlices := make([]v1alpha1.Slice, 0, capacity)
 
 	for _, psa := range wl.Status.Admission.PodSetAssignments {
+		podSet := podset.FindPodSetByName(wl.Spec.PodSets, psa.Name)
+		if !core.IsRelevantPodTemplateSpec(podSet.Template) {
```

Comment:
I think that we miss the case where the topology assignment is invalid (missing subblock level)

Can we define new function, `shouldCreateSliceForPodsetAssignment`
Then, in there we can check:
* IsRelevantPodTemplateSpec
* IsAssignmentValid(PodSetAssignment) (new function, in topology/validation.go)?

Reply:
Done